### PR TITLE
Updating to sdk's for release

### PR DESCRIPTION
**Pull request by mhoeger**
Updating nugets generated from azure-webjobs-sdk to be pulled into script for a prod release

### CODE REVIEW

**Comment by brettsam** [comment]
I checked in a CosmosDBTrigger fix today and *just* published it to MyGet. I saw the alert for this PR right after I hit "Deploy" -- so it probably went up after you looked.

**Comment by mhoeger** [comment]
Ahh, ok! Will do!

File: src/WebJobs.Script.Host/packages.config [hunk 1]
```diff
@@ -19,15 +19,15 @@
...
   <package id="Microsoft.Azure.WebJobs.Extensions.EventGrid" version="1.0.0-beta3-10015" targetFramework="net471" />
-  <package id="Microsoft.Azure.WebJobs.Extensions.MobileApps" version="1.1.0-beta4-10582" targetFramework="net471" />
-  <package id="Microsoft.Azure.WebJobs.Extensions.NotificationHubs" version="1.1.0-beta4-10582" targetFramework="net471" />
-  <package id="Microsoft.Azure.WebJobs.Extensions.SendGrid" version="2.1.0-beta4-10582" targetFramework="net471" />
-  <package id="Microsoft.Azure.WebJobs.Extensions.Twilio" version="1.1.0-beta4-10582" targetFramework="net471" />
+  <package id="Microsoft.Azure.WebJobs.Extensions.MobileApps" version="2.2.0-beta1-10588" targetFramework="net471" />
+  <package id="Microsoft.Azure.WebJobs.Extensions.NotificationHubs" version="2.2.0-beta1-10588" targetFramework="net471" />
+  <package id="Microsoft.Azure.WebJobs.Extensions.SendGrid" version="2.2.0-beta1-10588" targetFramework="net471" />
+  <package id="Microsoft.Azure.WebJobs.Extensions.Twilio" version="2.2.0-beta1-10588" targetFramework="net471" />
   <package id="Microsoft.Azure.WebJobs.Logging.ApplicationInsights" version="2.2.0-beta1-11229" targetFramework="net471" />
```
Comment (brettsam) [inline src/WebJobs.Script.Host/packages.config]: A few of these are incorrect. Make sure that each of the previous major versions still matches after the update. E.G. DocumentDB should be 1.2.0-beta1-10588.

File: src/WebJobs.Script.Host/WebJobs.Script.Host.csproj [hunk 2]
```diff
@@ -105,43 +105,43 @@
...
-      <HintPath>..\..\packages\Microsoft.Azure.WebJobs.Extensions.DocumentDB.1.1.0-beta4-10582\lib\net45\Microsoft.Azure.WebJobs.Extensions.DocumentDB.dll</HintPath>
+      <HintPath>..\..\packages\Microsoft.Azure.WebJobs.Extensions.DocumentDB.2.2.0-beta1-10588\lib\net45\Microsoft.Azure.WebJobs.Extensions.DocumentDB.dll</HintPath>
     </Reference>
     <Reference Include="Microsoft.Azure.WebJobs.Extensions.EventGrid, Version=1.0.0.0, Culture=neutral, processorArchitecture=MSIL">
       <HintPath>..\..\packages\Microsoft.Azure.WebJobs.Extensions.EventGrid.1.0.0-beta3-10015\lib\net46\Microsoft.Azure.WebJobs.Extensions.EventGrid.dll</HintPath>
     </Reference>
     <Reference Include="Microsoft.Azure.WebJobs.Extensions.MobileApps, Version=1.1.0.0, Culture=neutral, processorArchitecture=MSIL">
-      <HintPath>..\..\packages\Microsoft.Azure.WebJobs.Extensions.MobileApps.1.1.0-beta4-10582\lib\net45\Microsoft.Azure.WebJobs.Extensions.MobileApps.dll</HintPath>
+      <HintPath>..\..\packages\Microsoft.Azure.WebJobs.Extensions.MobileApps.2.2.0-beta1-10588\lib\net45\Microsoft.Azure.WebJobs.Extensions.MobileApps.dll</HintPath>
```
Comment (fabiocav) [inline src/WebJobs.Script.Host/WebJobs.Script.Host.csproj]: These are major version updates. We want to stick with the latest from the same major version release.

File: src/WebJobs.Script.Host/packages.config [hunk 3]
```diff
@@ -17,19 +17,19 @@
...
-  <package id="Microsoft.Azure.WebJobs.Core" version="2.2.0-beta1-11214" targetFramework="net471" />
-  <package id="Microsoft.Azure.WebJobs.Extensions" version="2.1.0-beta4-10582" targetFramework="net471" />
-  <package id="Microsoft.Azure.WebJobs.Extensions.ApiHub" version="1.0.0-beta6-10582" targetFramework="net471" />
+  <package id="Microsoft.Azure.WebJobs" version="2.2.0-beta1-11229" targetFramework="net471" />
+  <package id="Microsoft.Azure.WebJobs.Core" version="2.2.0-beta1-11229" targetFramework="net471" />
+  <package id="Microsoft.Azure.WebJobs.Extensions" version="2.2.0-beta1-10588" targetFramework="net471" />
+  <package id="Microsoft.Azure.WebJobs.Extensions.ApiHub" version="2.2.0-beta1-10588" targetFramework="net471" />
   <package id="Microsoft.Azure.WebJobs.Extensions.BotFramework" version="1.0.15-beta" targetFramework="net471" />
-  <package id="Microsoft.Azure.WebJobs.Extensions.DocumentDB" version="1.1.0-beta4-10582" targetFramework="net471" />
+  <package id="Microsoft.Azure.WebJobs.Extensions.DocumentDB" version="2.2.0-beta1-10588" targetFramework="net471" />
```
Comment (fabiocav) [inline src/WebJobs.Script.Host/packages.config]: Review major version updates.

**Comment by fabiocav** [comment]
One more... can you make sure Event Grid is updated to 1.0.0-beta3-10020

**Comment by mhoeger** [comment]
Thanks for catching the errors! I did a blanket find/replace and didn't notice that sdk-extensions nuget packages had a different major/minor version. Updated and added event grid update too

**Pull request merged**